packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/MultiItemMenu.tsx

```diff
@@ -220,7 +220,7 @@ export const MultiMenu = ({ selectedItems, page }: IMultiMenuProps) => {
     ? [{ label: 'Export Items', fn: exportItems }]
     : [];
 
-  const DELETE = { label: 'Delete Items', fn: deleteItems };
+  const DELETE = { label: 'Archive Items', fn: deleteItems };
```

Comment:
While we are here, can we sentence cases these labels? 😬

Reply:
Thought about that, but I would sentence case maybe the entire app in one PR, if we have time for that next week, as there will be a lot of changes in context menus and not only